### PR TITLE
PLAT-69025: Create initial welcome screen to start demo

### PR DESCRIPTION
**Pull request by germboy**
Added the initial welcome screen (`FullscreenPopup`) to start the demo. A user must be selected, then upon pressing the "Select" button, the welcome popup will hide and you can then navigate the app.

Per discussion with @Djspaceg, I have removed the user `SliderButton` from the `Settings` view, since it didn't apply there anymore. That was the only feature updating state via `AppContextConnect` on that view, so I have removed that as well. We'll ultimately want to hook up the settings to save state, so we may need to re-add it later in some (different) capacity.

### CODE REVIEW

File: console/src/components/WelcomePopup/WelcomePopup.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+	propTypes: {
+		onClose: PropTypes.func,
+		updateUser: PropTypes.func,
+		userId: PropTypes.number
+	},
+
+	render: ({onClose, updateUser, userId, ...rest}) => (
+		<FullscreenPopup {...rest} onClose={onClose}>
+			<Panels>
```
Comment (ryanjduffy) [inline console/src/components/WelcomePopup/WelcomePopup.js]: Is `<Panels>` necessary here for `<Panel>` to work or is this a placeholder for future panels?

File: console/src/components/WelcomePopup/WelcomePopup.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+				<Panel>
+					<Divider startSection>User Selection</Divider>
+					<Group
+						childComponent={SwitchItem}
+						select="radio"
+						selectedProp="selected"
+						defaultSelected={userId - 1}
```
Comment (ryanjduffy) [inline console/src/components/WelcomePopup/WelcomePopup.js]: Nit: alphabetize

File: console/src/components/WelcomePopup/WelcomePopup.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
+import Button from '@enact/agate/Button';
+import Divider from '@enact/agate/Divider';
+import kind from '@enact/core/kind';
+import FullscreenPopup from '@enact/agate/FullscreenPopup';
+import Group from '@enact/ui/Group';
+import {Panel, Panels} from '@enact/agate/Panels';
+import PropTypes from 'prop-types';
+import React from 'react';
+import SwitchItem from '@enact/agate/SwitchItem';
```
Comment (ryanjduffy) [inline console/src/components/WelcomePopup/WelcomePopup.js]: Nit: Consistent import ordering

File: console/src/components/WelcomePopup/WelcomePopup.js (hunk 1 above)
Comment (germboy) [reply console/src/components/WelcomePopup/WelcomePopup.js]: This is a placeholder. In a related feature that I'm working on, I'm adding additional panels which are the views for the welcome screen.

File: package.json [hunk 4]
```diff
@@ -8,5 +8,8 @@
   "repository": "https://github.com/enactjs/agate-apps",
   "eslintConfig": {
     "extends": "enact/strict"
+  },
+  "dependencies": {
+    "roslib": "^0.20.0"
```
Comment (Djspaceg) [inline package.json]: Is `roslib` a dependency of the "components" folder components? I didn't think so. This may need to be removed.

**Pull request merged**